test-suite/WD13-2.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>WSG Test: Progressive Enhancement</title>
+		<link rel="author" title="Alexander Dawson" href="https://alexanderdawson.com/">
+		<link rel="help" href="https://w3c.github.io/sustainableweb-wsg/#develop-a-mobile-first-layout">
+		<meta name="flags" content="">
+		<meta name="assert" content="Test Checks: Provide Progressive Enhancement Feature Testing Within Projects">
+		<style>
+			img { border: 1px solid black; }
+		</style>
+		<script>
+
+		</script>
+	</head>
+	<body>
+		<img src="assets/map.jpg" alt="This is a map to buried treasure!">
+		<p>Test passes if the page functions with JavaScript, CSS, and then assets like images and media disabled.</p>
```

Comment:
Trying to make it clearer that all should be disabled, maybe we could do something like this:

```suggestion
	<p>Test passes if the page functions when the user disables JavaScript, CSS, images and other media.</p>
```

Reply:
Noted, I'll ensure this is resolved in the next PR push.